src/components/common/PeripheralsDialog.vue

```diff
@@ -414,9 +415,9 @@ export default class ManualProbeDialog extends Mixins(StateMixin) {
           break
 
         case 'can':
-          for (const canbusInterface in this.canbusInterfaces) {
+          this.canbusInterfaces.forEach((canbusInterface) => {
             SocketActions.machinePeripheralsCanbus(canbusInterface)
-          }
+          })
```

Comment:
Plese revert this, I see no reason to use `.forEach` instead of a regular `for` cycle here.

Reply:
> Plese revert this, I see no reason to use `.forEach` instead of a regular `for` cycle here.

The const canbusInterface obtained directly by using for in is a serial number, not a real network device name.
You will receive an error from moonraker
![image](https://github.com/user-attachments/assets/ba8e45aa-bf02-437f-9d0e-93fddd7550f4)


Or can we do this?
```
for (const canbusInterface in this.canbusInterfaces) {
    SocketActions.machinePeripheralsCanbus(this.canbusInterfaces[canbusInterface])
}
```

Reply:
Ohhh, another mistake of mine!

That should be `of`, not `in`... this is how it should have been:

```typescript
for (const canbusInterface of this.canbusInterfaces)
```